src/core_modules/capture-core/components/ListView/Main/ListViewMain.component.js

```diff
@@ -80,11 +80,18 @@ class ListViewMainPlain extends React.PureComponent<Props> {
             filters,
             updatingWithDialog,
             onSelectRow,
+            customRowMenuContents,
             ...passOnProps
         } = this.props;
+
+        const ListComponent = (!customRowMenuContents || customRowMenuContents.length <= 0) ?
```

Comment:
I think in this case we can say or could an array length be `< 0`? 
```suggestion
        const ListComponent = (!customRowMenuContents || !customRowMenuContents.length) ?
```

Reply:
Changing. Old habbit.